Sources/Testing/Testing.docc/ParameterizedTesting.md

```diff
@@ -142,3 +142,9 @@ words, this test function will be passed the inputs `(.burger, 1)`,
 - ``Test(_:_:arguments:)-8kn7a``
 - ``Test(_:_:arguments:_:)``
 - ``Test(_:_:arguments:)-3rzok``
+- ``CustomTestArgumentEncodable``
+
+## See Also
+
+- ``Test/Parameter``
+- ``Test/Case``
```

Comment:
We should probably include `Test/Case/current` and `Test/Case/Argument` somewhere

Reply:
These are just links to the types. They're still categorized under `Test` in the DocC overview.